data_taking_scripts/dripline_logger.py

```diff
@@ -0,0 +1,29 @@
+from dripline.core import Interface
+import time
+class DriplineLogger:
```

Comment:
@parashar2399 The name of this class is way too generic. This class is pretty specific to logging S21 and S11 for a modemap measurement. 
https://github.com/driplineorg/dripline-python/blob/master/dripline/implementations/postgres_sensor_logger.py 
and
https://github.com/driplineorg/dripline-python/blob/master/dripline/implementations/postgres_interface.py
Are for more generic dripline logging.

Reply:
I was thinking that we could use it for all our data logging functions. But I can change it if you'd like. 
I'll go ahead and change the name of log_modemap for now